src/pages/messages/components/ExportNviStatusButton.tsx

```diff
@@ -0,0 +1,33 @@
+import FileDownloadOutlinedIcon from '@mui/icons-material/FileDownloadOutlined';
+import { LoadingButton } from '@mui/lab';
+import { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useFetchNviReportExport } from '../../../api/hooks/useFetchNviReportExport';
+import { dataTestId } from '../../../utils/dataTestIds';
+import { triggerFileDownload } from '../../../utils/download-file-helpers';
+import { useNviCandidatesParams } from '../../../utils/hooks/useNviCandidatesParams';
+
+export const ExportNviStatusButton = () => {
+  const { t } = useTranslation();
+  const { year } = useNviCandidatesParams();
+  const [fetchReport, setFetchReport] = useState(false);
+
+  const fetchNviApprovalReportQuery = useFetchNviReportExport(year, fetchReport, setFetchReport);
```

Comment:
vet ikke om det ble mye bedre egentlig, for min del ble det hvertfall vanskeligere nå å skjønne hva som foregår her, og hvorfor man må sende med `fetchReport` og `setFetchReport` som argument. Men komponenten ble hvertfall litt mindre, og kanskje ryddigere da

Reply:
Kan kanskje fetchReport og setFetchReport deklareres inne i hooken og så returneres setFetchReport fra hooken og brukes i komponenten?

Reply:
hmm, ja det høres mye bedre ut egentlig 🤔

Reply:
Syns det ser mye bedre ut ssånn ja! Langt mer intuitiv og lesbart kall til hooken 🙌